std/range/interfaces.d

```diff
@@ -208,6 +208,13 @@ interface RandomAccessInfinite(E) : ForwardRange!E {
 interface InputAssignable(E) : InputRange!E {
     ///
     @property void front(E newVal);
+
+    @property E front();
```

Comment:
The other members are already exposed because InputAssignable shows that it inherits from InputRange.